runtime/commonMain/src/kotlinx/serialization/internal/SerialClassDescImpl.kt

```diff
@@ -108,5 +103,10 @@ open class SerialClassDescImpl @JvmOverloads constructor(
         return result
     }
 
-    override fun toString() = "$name$names"
+    override fun toString(): String {
+        return indices.entries.joinToString(", ", "$name(", ")") { it.key + ": " + getElementDescriptor(it.value) }
```

Comment:
I don't think that including whole descriptor from `getElementDescriptor` to the String output is reasonable – it will recursively dump whole descriptor tree and bloat output. I think `getElementDescriptor(it.value).name` is sufficient